cli/src/main/java/com/devonfw/tools/ide/common/SystemPath.java

```diff
@@ -2,17 +2,14 @@
 
 import com.devonfw.tools.ide.context.IdeContext;
 import com.devonfw.tools.ide.os.SystemInfoImpl;
+import com.devonfw.tools.ide.os.WindowsPathSyntax;
 import com.devonfw.tools.ide.variable.IdeVariables;
 
 import java.io.File;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Avoid star import

Reply:
Thanks for the notice. I manually disabled this already multiple times. As I wrote in #52 we have a serious problem with these settings.
I will manually fix this somehow to complete the PR.